readme.md

```diff
@@ -0,0 +1,4 @@
+To install all dependencies run npm install
+to start server run npm start
+server will start at localhost:5000
+to search for querry use URL http://localhost:5000/search/QUERRY_HERER
```

Comment:
Please make it so its a REST GET request thats accepts JSON as a body argument. 
It should not require any autorization for now.
You should be able to use POSTMAN to test it.